pkg/e2e/openshift/suites/smoke_suite.go

```diff
@@ -364,7 +364,7 @@ func (tc *testConfig) ExpectNewNodeWhenDeletingMachine(ctx context.Context) erro
 	var triagedWorkerNode corev1.Node
 MachineLoop:
 	for _, m := range machineList.Items {
-		if m.Labels["sigs.k8s.io/cluster-api-machine-role"] == "worker" {
+		if m.Labels["machine.openshift.io/cluster-api-machine-role"] == "worker" {
```

Comment:
FYI: I dropped the label in ginkgo based testing in favor of `node-role.kubernetes.io/worker` label.